src/components/Community/Premium.tsx

```
@@ -0,0 +1,17 @@
+import { PremiumContainer } from "@/src/styles/Community.styled";
+import { BolText, Button, NorText } from "@/src/styles/GlobalStyles";
+import React from "react";
+import { GiCheckedShield } from "react-icons/gi";
+type PremiumProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)